docs/reference/cat/shards.asciidoc

```diff
@@ -269,6 +271,8 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=master-timeout]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-s]
 
+include::{docdir}/rest-api/common-parms.asciidoc[tag=time]
+
```

Comment:
i think this should also include bytes 
```
GET _cat/shards?h=sm&bytes=b
```

Reply:
It looks like `bytes` is already documented for this API:
https://github.com/elastic/elasticsearch/blame/be233f3d235b39d9315fc79974e9b69f20b1fe38/docs/reference/cat/shards.asciidoc#L29